### ISSUE TITLE
Raj092207

### ISSUE THREAD

**Issue by Julief8** [rmercer33/christmind.info#76]
Update:  fast forwarded and finished the timing and sent it.

Hi Rick,  I accidentally hit the stop instead of the pause button just before the spot below in the timing.  Being then unable to start it up again except at the start I saved the timing and sent it to you.  Can you complete the timing from that spot as it is very close to the end.  Also in this place there was a sentence in the highlighted area that wasn't said by Raj and had omit on it yet it appeared in the text.  I removed it in the attached file. Can the player be changed to allow someone to start where they left off in the timing in a case like this?

Thanks,  Julie

PS: I sent a request in Slack to reserve the rest of 2007.  Did you get that and is it reserved?

[092207.md.zip](https://github.com/rmercer33/christmind.info/files/1443668/092207.md.zip)

--------------------------------------------------------
<div markdown="1" class="well book">
God waits your witness to His Son and to Himself.
</div>

God waits your witness to your Sonship and His Fathership.

**Comment by rmercer33** [comment]
Julie, Yes, I can complete the timing but it is really hard to make it easy to do - if you know what I mean. So, for now anyway, I'd like to leave it as it is.

Regarding {: .omit} - it doesn't remove text from the page but it does prevent the 'bullseye' icon from being displayed thus preventing timing for an omitted paragraph from being collected. So we don't want to remove it from the file. So, I will ignore the changes you made to the transcript unless you made other changes. Please let me know if that's the case.

I will reserve the rest of 2007 for you.

**Comment by Julief8** [comment]
Hi Rick

I used the fast forward and finished the timing so the second one I submitted should be complete. Yes there was one other change in that file besides the “omit”. He repeated himself on “When you have seen your brother as yourself you will be RELEASED to knowledge.” It’s the first ACIM quote in the file. I just repeated it like he said it. 

Banyak cinta, 💖. Julie

PS: Leaving for Bali in 4 days. Yeah 🎉
